roles/kubernetes-apps/cloud_controller/oci/tasks/credentials-check.yml

```
@@ -3,31 +3,31 @@
 - name: "OCI Cloud Controller | Credentials Check | oci_private_key"
   fail:
     msg: "oci_private_key is missing"
-  when: (oci_use_instance_principals == false) and
+  when: (not oci_use_instance_principals) and
```

These checks in general can also be written as a list instead:

```suggestion
  when:
    - not oci_use_instance_principals
    - oci_private_key is not defined or oci_private_key == ""
```

---

Yes we could improve readability. Maybe in another PR?